src/incus.js

```diff
@@ -1,3 +1,5 @@
+var ee = require('event-emitter');
```

Comment:
ee seems a bit vague. emitter seems like it would be clearer...butttt I am not a stickler cuz it is equally weird to do emitter.emit